utils/calc_total_unclaimed_fees.go

```diff
@@ -0,0 +1,61 @@
+package utils
+
+import (
+	"log"
+	"math/big"
+	"strconv"
+)
+
+var tokenSymbolToName = map[string]string{
+	"WETH": "ethereum",
+	"DAI":  "dai",
+	"UNI":  "uniswap",
+}
```

Comment:
Is this fetchable from the CoinGecko API?

Reply:
I don't think so, that's why I hardcoded